install_config/syncing_groups_with_ldap.adoc

```diff
@@ -860,3 +867,173 @@ users: <5>
+The *otheradmins* group has one user member: *Jim*.  This example explains:
+
+* How the group and users are added to the LDAP server.
+* What the LDAP sync configuration file looks like.
+* What the resulting Group record in {product-title} will be after synchronization.
+
+In the augmented Active Directory schema, both users (Jane and Jim) and groups
+exist in the LDAP server as first-class entries, and group membership is stored
+in attributes on the user or the group. The following snippet of `ldif` defines the
+users and group for this schema:
+
+.LDAP Entries Using Augmented Active Directory Schema With Recursive Members: *_augmented_active_directory_recursive.ldif_*
+====
+[source,ldif]
+----
+dn: ou=users,dc=example,dc=com
+objectClass: organizationalUnit
+ou: users
+
+dn: cn=Jane,ou=users,dc=example,dc=com
+objectClass: person
+objectClass: organizationalPerson
+objectClass: inetOrgPerson
+objectClass: testPerson
+cn: Jane
+sn: Smith
+displayName: Jane Smith
+mail: jane.smith@example.com
+memberOf: cn=admins,ou=groups,dc=example,dc=com <1>
+
+dn: cn=Jim,ou=users,dc=example,dc=com
+objectClass: person
+objectClass: organizationalPerson
+objectClass: inetOrgPerson
+objectClass: testPerson
+cn: Jim
+sn: Adams
+displayName: Jim Adams
+mail: jim.adams@example.com
+memberOf: cn=otheradmins,ou=groups,dc=example,dc=com <1>
+
+dn: ou=groups,dc=example,dc=com
+objectClass: organizationalUnit
+ou: groups
+
+dn: cn=admins,ou=groups,dc=example,dc=com <2>
+objectClass: group
+cn: admins
+owner: cn=admin,dc=example,dc=com
+description: System Administrators
+member: cn=Jane,ou=users,dc=example,dc=com
+member: cn=otheradmins,ou=groups,dc=example,dc=com
+
+dn: cn=otheradmins,ou=groups,dc=example,dc=com <2>
+objectClass: group
+cn: otheradmins
+owner: cn=admin,dc=example,dc=com
+description: Other System Administrators
+memberOf: cn=admins,ou=groups,dc=example,dc=com <1> <3>
+member: cn=Jim,ou=users,dc=example,dc=com
+----
+<1> The user's and group's memberships are listed as attributes on the object.
+<2> The groups are first-class entries on the LDAP server.
+<3> The *otheradmins* group is a member of the *admins* group.
+====
+
+To sync this group, you must first create the configuration file. The
+augmented Active Directory with recursive members schema requires you to
+provide an LDAP query definition for user entries and the attributes
+with which to represent them in the internal {product-title} Group records.
+Furthermore, the `oadm groups sync` command must explicitly
+xref:../install_config/syncing_groups_with_ldap.adoc#running-ldap-sync[whitelist]
+groups and the user's `groupMembershipAttributes` must specify
+https://msdn.microsoft.com/en-us/library/aa746475(v=vs.85).aspx[LDAP_MATCHING_RULE_IN_CHAIN].
+`groupUIDAttribute` must be set to `dn` and the `groupsQuery` must not set `filter`.
+`groupsQuery` must set a valid `derefAliases` and should not set `baseDN` or `scope`
+as those values are ignored.
```

Comment:
Right, but these are not examples, they are imperative statements. You can't have an example that tells me `baseDN` is ignored.